client/src/components/Sle/Sle.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import "./Sle.css";
+import "./Sidebar.css";
+import "./Sidebar.jsx";
+import Sidebar from "./Sidebar";
+
+
+
+function Sle(props) {
+    return (
+    
+    <div>
+    
+    <div class = "line">
+    </div>
+    
+    <div class = "sidebar">
```

Comment:
The sidebar component seems misnamed. It looks to me like it's what tab on the sidebar, right? That component should probably be named sidebarElement, then we can create a new component for the entire sidebar.